sample/sample_test.go

```diff
@@ -0,0 +1,37 @@
+package sample
+
+import (
+	"io/ioutil"
+	"reflect"
+	"testing"
+)
+
+func TestSampleTargetHaveAllFiles(t *testing.T) {
+    data_files := listFilesFromDir("/home/alexandre/Desenvolvimento/minha-receita/data")
+    sample_files := listFilesFromDir("/home/alexandre/Desenvolvimento/minha-receita/sample_data")
+    assertArraysHaveSameItems(t, data_files, sample_files)
+}
+
+func listFilesFromDir(dir string) []string {
+    var files []string
+    paths, err := ioutil.ReadDir(dir)
+    if err != nil {
+        panic(err)
+    }
+
+    for _, file := range paths {
+        if ! file.IsDir() {
+            files = append(files, file.Name())
+        }
+    }
+    return files
+}
+
+func assertArraysHaveSameItems(t *testing.T, a, b []string) {
+        if len(a) != len(b) {
+            t.Errorf("Arrays have different lengths")
+        }
+        if ! reflect.DeepEqual(a, b) {
+            t.Errorf("File not found in sample")
+        }
```

Comment:
A gente precisa checar as duas coisas (`len` e `DeepEqual`)? Digo isso pois acho que o `DeepEqual` já retornaria falso caso o tamanho seja diferente… e, se for esse o caso, podemos matar o `assertArraysHaveSameItems` e usar o `DeepEqual` direto na função princiapl do teste.